templates/docker/updateos_ubi7

```diff
@@ -12,18 +12,23 @@ RUN yum install -y ${RAPIDS_SRC_DIR}/tmp/*.rpm && \
     yum upgrade -y && \
     yum install -y \
       bzip2 \
+      clang \
       curl \
+      file \
       git \
+      make \
       screen \
       vim \
       wget \
       which \
-      clang \
-      make \
       libnccl-2.4.2-1+cuda${CUDA_MAJORMINOR_VERSION} \
       libnccl-devel-2.4.2-1+cuda${CUDA_MAJORMINOR_VERSION} \
-      libnccl-static-2.4.2-1+cuda${CUDA_MAJORMINOR_VERSION} \
-      gmp-devel mpfr-devel libmpc-devel file 
+      libnccl-static-2.4.2-1+cuda${CUDA_MAJORMINOR_VERSION} && \
```

Comment:
TODO check if these libraries really needed if we do not build GCC but use the one provided by devtoolset